alpenhorn/io/nearline.py

```diff
@@ -0,0 +1,408 @@
+"""Cedar nearline I/O.
+
+These I/O classes are specific to the nearline tape archive at cedar.
```

Comment:
It's debatable, but I wonder if this should go in `alpenhorn-chime` instead?

Reply:
That's not a bad point...

Reply:
There's nothiing _explicity_ CHIME or cedar specific here; this should work on any HSM-backed lustre filesystem.  Maybe the thing to do is change this description.